cms/djangoapps/contentstore/views/component.py

```diff
@@ -141,6 +142,35 @@ def container_handler(request, usage_key_string):  # pylint: disable=too-many-st
         return HttpResponseBadRequest("Only supports HTML requests")
 
 
+@require_GET
+@login_required
```

Comment:
It seems like decorators that are required to use this view in an Iframe are missing, please add `xframe_options_exempt` and `ensure_csrf_cookie` as it is done in LMS: https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/courseware/views/views.py#L1546.

Reply:
`xframe_options_exempt` decorator added.